### ISSUE TITLE
Could not install emacs-plus@28 - expected value in expression

### ISSUE THREAD

**Issue by memeplex** [d12frosted/homebrew-emacs-plus#403]
Please make sure to follow these steps (and mark the checkboxes):

- [x] run `brew update` and try to reproduce the issue again
- [x] run `brew doctor`, fix all issues and try to reproduce your issue again
- [x] run `brew config` and `brew doctor` and include their output

## What happened (include command output)

<pre>
$ brew install emacs-plus  --with-nobu417-big-sur-icon
</pre>

<details>
  <summary>Command output</summary>
  <pre>
==> Cloning https://github.com/emacs-mirror/emacs.git
Cloning into '/Users/carlos/Library/Caches/Homebrew/emacs-plus@28--git'...
Updating files: 100% (4490/4490), done.
==> Checking out branch emacs-28
Already on 'emacs-28'
Your branch is up to date with 'origin/emacs-28'.
==> Installing emacs-plus@28 from d12frosted/emacs-plus
==> Patching
==> Applying fix-window-role.patch
patching file src/nsterm.m
Hunk #1 succeeded at 8577 (offset -191 lines).
==> Applying system-appearance.patch
patching file src/frame.h
patching file src/nsfns.m
patching file src/nsterm.m
Hunk #1 succeeded at 1878 (offset -11 lines).
Hunk #2 succeeded at 5384 (offset -11 lines).
Hunk #3 succeeded at 5630 (offset -11 lines).
Hunk #4 succeeded at 5680 (offset -11 lines).
Hunk #5 succeeded at 5906 (offset -11 lines).
Hunk #6 succeeded at 8910 (offset 8 lines).
Hunk #7 succeeded at 10066 (offset 8 lines).
==> ImageMagick PKG_CONFIG_PATH:
/usr/local/opt/imagemagick/lib/pkgconfig
==> ./autogen.sh
==> ./configure --enable-locallisppath=/usr/local/share/emacs/site-lisp --infodir=/usr/local/Cellar/emacs-plus@28
==> make
Last 15 lines from /Users/carlos/Library/Logs/Homebrew/emacs-plus@28/03.make:
clang -c    -MMD -MF deps/fingerprint.d -MP -Wno-switch -Wno-pointer-sign -Wno-string-plus-int -Wno-unknown-attributes -Wno-initializer-overrides -Wno-tautological-compare -Wno-tautological-constant-out-of-range-compare   -g3 -O2 -I. -I../src -I. -I./../src  -o fingerprint.o fingerprint.c
clang -c    -MMD -MF deps/acl_entries.d -MP -Wno-switch -Wno-pointer-sign -Wno-string-plus-int -Wno-unknown-attributes -Wno-initializer-overrides -Wno-tautological-compare -Wno-tautological-constant-out-of-range-compare   -g3 -O2 -I. -I../src -I. -I./../src  -o acl_entries.o acl_entries.c
In file included from acl_entries.c:22:
In file included from ./acl-internal.h:23:
In file included from ./stdlib.h:36:
In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/stdlib.h:66:
In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/sys/wait.h:110:
In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk/usr/include/sys/resource.h:72:
./stdint.h:89:5: error: expected value in expression
#if
    ^
1 error generated.
make[1]: *** [acl_entries.o] Error 1
make: *** [lib] Error 2
make: *** Waiting for unfinished jobs....

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/d12frosted/homebrew-emacs-plus/issues

These open issues may also help:
Need to set LIBRARY_PATH for emacs-plus@28 with --with-native-comp https://github.com/d12frosted/homebrew-emacs-plus/issues/378
Permission issues with emacs-plus running as a service https://github.com/d12frosted/homebrew-emacs-plus/issues/304
Emacs-plus@28 with native comp crashes on MacOS 12/Monterey https://github.com/d12frosted/homebrew-emacs-plus/issues/361
  </pre>
</details>

## Output of `brew config`

<pre>
$ brew config
HOMEBREW_VERSION: 3.2.14-76-g28e4c9c
ORIGIN: https://github.com/Homebrew/brew
HEAD: 28e4c9c5d9f8bd66a3850ad07f60cfae1cc0b8e8
Last commit: 7 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 02faa829ade344e48223e218dc333a07f4c63a07
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.3 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 13.0.0 build 1300
Git: 2.33.0 => /usr/local/bin/git
Curl: 7.64.1 => /usr/local/Homebrew/Library/Homebrew/shims/shared/curl
macOS: 11.6-x86_64
CLT: 13.0.0.0.1.1630607135
Xcode: N/A
</pre>

## Output of `brew doctor`

<pre>
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /usr/local/Caskroom/miniconda/base/bin/python3-config
  /usr/local/Caskroom/miniconda/base/bin/ncursesw6-config
  /usr/local/Caskroom/miniconda/base/bin/python3.8-config
  /Users/carlos/Base/Environments/Default/bin/pre-commit-validate-config

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> /Users/carlos/.bash_profile
</pre>

**Comment by memeplex** [comment]
Notice I've installed the same package with the same trivial option in the same computer without any issue.

**Comment by d12frosted** [comment]
Yup, the build is dead https://github.com/d12frosted/homebrew-emacs-plus/runs/3799188873?check_suite_focus=true

Unrelated to icon :) Let me rephrase your issue.

**Comment by memeplex** [comment]
Working again, whether it was you or upstream, thank you for being responsive!
